### ISSUE TITLE
mxSE improve error message when object not found

### ISSUE THREAD

**Issue by tbates** [OpenMx/OpenMx#180]
Asking `mxSE` for a parameter/object that doesn't exist gives an obscure error:

repro:

```R
m1 = mxModel("test", type="RAM",
	manifestVars = "X",
	mxPath(from="X", to = "X", arrows=2, lbound=0, labels= "X"),
	mxData(mData, type="cov", numObs = 10)
)
m1 = mxRun(m1)

mxSE("Iamtheonewhoknocks", m1)
```
> Error in as.vector(x, mode) : 
  cannot coerce type 'symbol' to vector of type 'any'
Error in mxSE("Iamtheonewhoknocks", m1) : 
  I had a problem evaluating your expression. Check that mxEval works for it.
Recall that elements of submodels are addressed as submodelName.objectName
For example, 'sub1.bob' refers to the object 'bob' in the submodel named 'sub1'.

Be nicer to 
1. not mention as.vector
2. say what to do with mxEval, because this works fine:
```R
mxEval("Iamtheonewhoknocks", m1)
     [,1]                
[1,] "Iamtheonewhoknocks"
```
3. technically, I guess, we don't need to tell people about algebras and submodels in models that don't contain any...

But perhaps something like

> Error in mxSE("Iamtheonewhoknocks", m1) :
Cannot find "Iamtheonewhoknocks" as a parameter or algebra in your model "test"
> Extra information:  If mxEval(Iamtheonewhoknocks, m1) doesn't work, you might have mis-spelled the parameter.
If you want to compute an SE on an parameter in a submodel, you need to include the submodes name: e.g. 'sub1.bob' to refer to the parameter(s) 'bob' in the 'bob' submodel.

**Comment by tbates** [comment]
FYI, this is generating a different error now, telling the user about "ontheflyalgebra", which can't work in `mxEval`

> Treating first argument as a character
Error in as.vector(x, mode) : 
  cannot coerce type 'symbol' to vector of type 'any'
Error in mxSE("Iamtheonewhoknocks", m1) : 
  Couldn't evaluate expression 'onTheFlyAlgebra'. Might help to check if it works in mxEval.
Recall also that elements of submodels are addressed as submodelName.objectName
For example, to refer to an object called 'bob' in submodel 'sub1', you would say 'sub1.bob'

**Comment by RMKirkpatrick** [comment]
What is the current status of this issue?  BTW, @tbates , the syntax in your OP doesn't provide a reproducible instance, since `mData` is nowhere defined.  Also, `mxEval("Iamtheonewhoknocks", m1)` will always trivially return a 1x1 matrix containing "iamtheonewhoknocks"; you probably meant to use `mxEvalByName()` instead.

**Comment by tbates** [comment]
seems unchanged under OpenMx version: 2.16.0.23 [GIT v2.16.0-23-g613f0c0]

```R
m1 = mxModel("test", type="RAM", manifestVars = "mpg",
	mxPath(from="mpg", arrows=2),
	mxData(cov(mtcars[,"mpg", drop=F]), type="cov", numObs = 10)
)
m1 = mxRun(m1)
mxEval("I_dont_exist", m1)
#    [,1]          
# [1,] "I_dont_exist"
```

and:

```R 
mxSE("I_dont_exist", m1)
```

> Treating first argument as a character
Error in as.vector(x, mode) : 
  cannot coerce type 'symbol' to vector of type 'any'
Error in mxSE("Iamtheonewhoknocks", m1) : 
  Couldn't evaluate expression 'onTheFlyAlgebra'. Might help to check if it works in mxEval.
Recall also that elements of submodels are addressed as submodelName.objectName
For example, to refer to an object called 'bob' in submodel 'sub1', you would say 'sub1.bob'.


Desireable to not refer to onTheFlyAlgebra (as this is internal) and perhaps refer to `mxEvalByName` instead of mxEval?

**Comment by RMKirkpatrick** [comment]
Closed with 749de5f .
